[['trace', 0, 0, 1, 0], ['self', 0, 1, 1, 1], ['setLog', 0, 2]]
['code603', 'code604']
error: 0.000185734106615265, n: 7, k: 1, d: 3, KS:1, rowW: 6, colW: 6

[['trace', 0, 2, 1, 0], ['trace', 0, 5, 2, 2], ['setLog', 0, 0]]
['code603', 'code604', 'code603']
error: 0.000458398199747023, n: 13, k: 1, d: 3, KS:1, rowW: 10, colW: 7

